migrations/add-filter.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  async up(db) {
+    let Filter = require("../node_modules/bad-words");
+    const filter = new Filter({emptyList: true});
+    filter.addWords("covid-19","coronavirus", "pandemic","masks","mask")
+    const articles = await db.collection('articles').find({}).toArray();
+    articles.map( async (article) => {
+        await db.collection('articles').updateOne({_id: article._id}, {$set: {isFiltered: filter.isProfane(article.title)}});
+        return article;
+    });
+  },
+
+  async down(db) {
```

Comment:
what does this function do? a little confused.. maybe more descriptive name and also a docstring ?

Reply:
This is an autogenerated file from migrate-mongo that reverts the changes to the mongo database. I probably can't change its name but I will add a docstring.